FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -99,34 +99,31 @@ final class ComposeViewController: TableNodeViewController {
         photosManager: PhotosManagerType = PhotosManager(),
         keyMethods: KeyMethodsType = KeyMethods()
     ) {
-        self.appContext = appContext
-        guard let email = appContext.dataService.email else {
-            fatalError("missing current user email") // todo - need a more elegant solution
-        }
-        self.email = email
+        self.userContext = userContext
+        self.email = userContext.user.email
         self.notificationCenter = notificationCenter
         self.input = input
         self.decorator = decorator
-        let clientConfiguration = appContext.clientConfigurationService.getSaved(for: email)
+        let clientConfiguration = userContext.clientConfigurationService.getSaved(for: userContext.user.email)
         self.contactsService = contactsService ?? ContactsService(
             localContactsProvider: LocalContactsProvider(
-                encryptedStorage: appContext.encryptedStorage
+                encryptedStorage: userContext.encryptedStorage
             ),
             clientConfiguration: clientConfiguration
         )
         let cloudContactProvider = cloudContactProvider ?? UserContactsProvider(
             userService: GoogleUserService(
-                currentUserEmail: email,
+                currentUserEmail: userContext.user.email,
                 appDelegateGoogleSessionContainer: UIApplication.shared.delegate as? AppDelegate
             )
         )
         self.cloudContactProvider = cloudContactProvider
         self.composeMessageService = composeMessageService ?? ComposeMessageService(
             clientConfiguration: clientConfiguration,
-            encryptedStorage: appContext.encryptedStorage,
-            messageGateway: appContext.getRequiredMailProvider().messageSender,
-            passPhraseService: appContext.passPhraseService,
-            sender: email
+            encryptedStorage: userContext.encryptedStorage,
```

Comment:
let's always call it `appContext` for consistency, even if type is either `AppContext` or `AppContextWithUser` (usually)